src/components/modals/UpdateModal.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Text, TouchableHighlight, View } from 'react-native'
+import { Image } from 'react-native-animatable'
+
+import edgeLogo from '../../assets/images/edgeLogo/Edge_logo_Icon.png'
+import s from '../../locales/strings.js'
+import { PrimaryButton } from '../../modules/UI/components/Buttons'
+import { LadderLayout } from '../common/LadderLayout'
+import { type AirshipBridge, AirshipModal, dayText, IconCircle, textSize, THEME } from './modalParts.js'
+
+type Props = {
+  bridge: AirshipBridge<mixed>,
+  newVersion: string,
+  released: string,
+  onUpdate(): void,
+  onSkip(): void
+}
+
+export class UpdateModal extends Component<Props> {
+  render () {
+    const { bridge } = this.props
+
+    return (
+      <AirshipModal bridge={bridge} onCancel={() => bridge.resolve()}>
+        <IconCircle>
+          <View
+            style={{
+              width: '100%',
+              height: '100%',
+              flex: 1,
+              alignItems: 'center',
+              justifyContent: 'center',
+              backgroundColor: THEME.COLORS.PRIMARY
+            }}
+          >
+            <Image source={edgeLogo} />
+          </View>
+        </IconCircle>
+
+        <View
+          style={{
+            padding: THEME.rem(1),
```

Comment:
Since the ladder layout already adds 1rem all around, this means we end up with 2rem on the outside. Is that intentional?